package/Editor/Settings/PluginSettings.cs

```diff
@@ -12,21 +12,32 @@ public class PluginSettings : EditorWindow
         #region Public Properties
 
         public static string ApiUrl => "https://api.cloud.scenario.com/v1";
+        public static string WebAppUrl { get { return webAppUrl; } }
         public static Preset TexturePreset { get { return GetPreset(EditorPrefs.GetString("scenario/texturePreset")); } }
         public static Preset SpritePreset { get { return GetPreset(EditorPrefs.GetString("scenario/spritePreset")); } }
         public static Preset TilePreset { get { return GetPreset(EditorPrefs.GetString("scenario/tilePreset")); } }
+        public static string TeamIdKey { get 
+            {
+                if (string.IsNullOrEmpty(teamIdKey))
+                {
+                    teamIdKey = EditorPrefs.GetString("TeamIdKey");
```

Comment:
can you save the string in "scenario/teamIdKey" like the other editor prefs plz ?

Reply:
Oh ! I miss this !

Reply:
> can you save the string in "scenario/teamIdKey" like the other editor prefs plz ?

That mean I changed other one

![image](https://github.com/scenario-labs/Scenario-Unity/assets/30622829/4380f451-c462-4114-96a9-03bd19d10c09)